xla/service/gpu/ir_emission_utils.h

```diff
@@ -88,6 +88,18 @@ inline constexpr absl::string_view kUncompilableFusion =
 
 inline constexpr absl::string_view kTopKCustomCallTarget = "__gpu$TopK";
 
+inline constexpr absl::string_view
```

Comment:
Please add brief documentation on what these new constant represents (see above for `__triton` fusions for example).